test/models/order_test.rb

```diff
@@ -35,6 +36,20 @@ class OrderTest < ActiveSupport::TestCase
     assert_equal 'ordered', user.orders.last.current_status
   end
 
+  test 'an order is saved with a total price' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.